Workaround some ghc-7.4 problem.
I think ghc-7.4 inlines something too aggressively.

Without this patch ./setup install phase fails with an
'Array.(!) is undefined value' error.
diff --git a/src/runtime/haskell/Data/Binary.hs b/src/runtime/haskell/Data/Binary.hs
index 2bebaf1..c1bee2c 100644
--- a/src/runtime/haskell/Data/Binary.hs
+++ b/src/runtime/haskell/Data/Binary.hs
@@ -1,4 +1,5 @@
 {-# LANGUAGE CPP, FlexibleInstances, FlexibleContexts #-}
+{-# OPTIONS_GHC -O0 #-}
 -----------------------------------------------------------------------------
 -- |
 -- Module      : Data.Binary
